### PR TITLE
Readying for 1.5.5 and migrating to the new csproj format

### PR DESCRIPTION
**Pull request by strobel1ght**

### CODE REVIEW

**Comment by Annosz** [comment]
Thanks!

File: UIInfoSuite2/UIInfoSuite2.csproj [hunk 1]
```diff
@@ -1,128 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <DeployModFolderName>$(MSBuildProjectName)</DeployModFolderName>
-    <TargetFrameworkProfile />
+    <Version>2.1.5</Version>
+    <TargetFramework>net5.0</TargetFramework>
+    <Platforms>AnyCPU</Platforms>
```
Comment (Deflaktor) [inline UIInfoSuite2/UIInfoSuite2.csproj:5]: Platforms and PlatformTarget properties can be removed

File: UIInfoSuite2/manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿{
     "Name": "UI Info Suite 2",
     "Author": "Annosz",
-    "Version": "2.1.4",
+    "Version": "2.1.5-beta",
     "Description": "Adds a useful information to the user interface. Based on Cdaragorn's excellent UI Info Suite.",
     "UniqueID": "Annosz.UiInfoSuite2",
     "EntryDll": "UIInfoSuite2.dll",
```
Comment (Deflaktor) [inline UIInfoSuite2/manifest.json:7]: Increase MinimumApiVersion from 3.0.0 to 3.13.0

File: UIInfoSuite2/UIInfoSuite2.csproj [hunk 3]
```diff
@@ -1,128 +1,12 @@
...
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DebugSymbols>true</DebugSymbols>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
```
Comment (Deflaktor) [inline UIInfoSuite2/UIInfoSuite2.csproj:37]: Since the x86 TargetProfile has been removed, it needs to be removed from the UIInfoSuite2.sln file as well

**Review by Deflaktor — CHANGES_REQUESTED** [review]

**Comment by Annosz** [comment]
Thanks for the work! Now that the 1.5.5 version is out, I implemented the changes, but sadly, I had to make a new pull request, because this one already had some merge conflicts. But your work showed me what I need to do, so huge thanks! :)

**Comment by Zamiell** [comment]
I just wanted to say thank you Ishan and Annosz for all of your work!

**Pull request closed without merging**